lib/native/binding.d.ts

```diff
@@ -118,6 +118,13 @@ export function mqtt_client_connection_new(
     client: NativeHandle,
     on_interrupted?: (error_code: number) => void,
     on_resumed?: (return_code: number, session_present: boolean) => void,
+    tls_ctx?: NativeHandle,
+    will?: { topic: StringLike, payload: String | Object | DataView, qos: number, retain: boolean },
+    username?: StringLike,
+    password?: StringLike,
+    use_websocket?: boolean,
+    proxy_options?: NativeHandle,
+    websocket_handshake_transform?: (request: any, done: (error_code?: number) => void) => void,
```

Comment:
Upon reflection and reading this again after some time removed, it feels like these function params should be typed, shouldn't they?

Reply:
did